t/client.t

```diff
@@ -133,3 +137,78 @@ GET /t
 .*replicas.*
 --- no_error_log
 [error]
+
+
+
+=== TEST 4: simple fetch sasl
+--- http_config eval: $::HttpConfig
+--- config
+    location /t {
+        content_by_lua '
+
+            local cjson = require "cjson"
+            local client = require "resty.kafka.client"
+
+            local broker_list = {
+                { host = "$TEST_NGINX_KAFKA_HOST", port = $TEST_NGINX_KAFKA_SASL_PORT ,
+                  sasl_config = { mechanism="PLAIN", user="$TEST_NGINX_KAFKA_SASL_USER", password = "$TEST_NGINX_KAFKA_SASL_PWD" },},
+            }
+
+            local messages = {
+                "halo world",
+            }
+
+            local cli = client:new(broker_list)
+
+            local brokers, partitions = cli:fetch_metadata("test")
+            if not brokers then
+                ngx.say("fetch err:", partitions)
+                return
+            end
+
+            ngx.say(cjson.encode(partitions))
+        ';
+    }
+--- request
+GET /t
+--- response_body_like
+.*replicas.*
+--- no_error_log
+[error]
+
+
+
+
```

Comment:
style: need three blank lines here.
please to `reindex` tool to format the test file automatically:
https://github.com/openresty/openresty-devel-utils#reindex